nlptest/modelhandler/jsl_modelhandler.py

```diff
@@ -397,3 +404,105 @@ def predict_raw(self, text: str) -> List[str]:
     def __call__(self, text: str) -> SequenceClassificationOutput:
         """Alias of the 'predict' method"""
         return self.predict(text=text)
+
+class PretrainedModelForTranslation(_ModelHandler):
+
+    """"""
+
+    def __init__(self, model) -> None:
+        
+        if model.__class__.__name__ == 'PipelineModel':
+            model = model
```

Comment:
Not relevant here but we should create a `PretrainedJSLModel` and factorize the logic within the constructor (noting it somewhere)

Reply:
**PretrainedJSLModel** is not possible right now, bcoz in the modelhandler, PretrainedModel is routing according to the module like **JSL -> jsl_modelhandler.py**. So, within that module have pretrainedNerModel, ..... etc same classes for all hubs.

Reply:
Yeah definitely, what I'm saying is since the `__init__` of all the `jsl_modelhandler.PretrainedModel***` is the same for all classes we could have a `jsl_modelhandler.BasePretrainedModel` and have all the `jsl_modelhandler.PretrainedModel***` extend that class.
Know what I mean?

Reply:
Yes, I got it.